app/components/windows/Troubleshooter.vue.ts

```diff
@@ -26,8 +28,16 @@ export default class Troubleshooter extends Vue {
 
   issueCode = this.windowsService.getChildWindowQueryParams().issueCode as TIssueCode;
 
+  private subscription: Subscription;
+
   mounted() {
     this.getSettings();
+    this.subscription = this.streamingService.streamingStatusChange
+      .pipe(
+        debounceTime(500),
+        tap(this.getSettings),
```

Comment:
What's the purpose of `tap` here?  Just to be explicit about the fact that we're calling `getSettings` for the side effect and not for the return value?

Reply:
Exactly.